src/libs/OptionsListUtils.ts

```diff
@@ -1931,8 +1939,18 @@ function getOptions(
             return;
         }
 
-        if ((!accountIDs || accountIDs.length === 0) && !isChatRoom) {
-            return;
+        // Reports that aren't searchable via participant won't have any participants returned with SearchForReports
+        // so we don't need to check if their participants exist in order for them to qualify as a valid option.
```

Comment:
It would be great if we could also explain:

- Which reports "are not searchable by participant" and why we do this.
- Clarify that these reports _might_ still have participants, but that we omit them from `SearchForReports` as a performance optimization.

Reply:
Done